rosetta/cosmos/launchpad/account.go

```diff
@@ -2,10 +2,60 @@ package launchpad
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
 	"github.com/coinbase/rosetta-sdk-go/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/tendermint/cosmos-rosetta-gateway/rosetta"
 )
 
+const AccountSdkHandler = "bank/balances/"
+
 func (l Launchpad) AccountBalance(ctx context.Context, request *types.AccountBalanceRequest) (
 	*types.AccountBalanceResponse, *types.Error) {
-	panic("implement me")
+
+	get, err := http.Get(fmt.Sprintf("%s%s%s", l.endpoint, AccountSdkHandler, request.AccountIdentifier.Address))
+	if err != nil {
+		return nil, rosetta.NewError(1, "error getting data from node")
+	}
+
+	body := get.Body
+	defer body.Close()
+
+	all, _ := ioutil.ReadAll(body)
```

Comment:
probably better to make an error check here since the `io.Reader` hits the network